blocks/footer/footer.css

```diff
@@ -1,147 +1,352 @@
 
-footer {
-  --footer-font-color: #fff;
-  
-  background-color: #202020;
-}
+   footer {
+    --footer-font-color: #fff;
+ 
+    background-color: #202020;
+  }
 
-footer .footer {
-  max-width: 70rem;
-  max-height: 36.75rem;
-  margin: 0 auto;
-  font-size: var(--body-font-size-s);
-}
+  footer .footer-links>div{
+    display: flex;
+    padding-bottom: 1.5rem;
+    border-bottom: 0.1px solid  #636161;
+  }
 
-footer .footer-links>div{
-  display: flex;
-  padding-bottom: 1.5rem;
-  border-bottom: 0.1px solid  #636161;
-}
+  footer .footer > div > div {
+    padding-top: 6rem;
+    padding-bottom: 1rem;
+  }
 
-footer .footer > div > div {
-  padding-top: 6rem;
-  padding-bottom: 1rem;
-}
+  footer .footer-links > div > div{
+    flex: 1 1 13.7875rem;
```

Comment:
when using all 3 values of the flex shorthand, I'd probably expand it to 3 declarations (grow, shrink, basis) just for clarity